.github/workflows/ci.yaml

```diff
@@ -87,6 +87,17 @@ jobs:
           name: ruff
           path: target/debug/ruff
 
+  cargo-fuzz:
+    runs-on: ubuntu-latest
+    name: "cargo fuzz"
+    steps:
+      - uses: actions/checkout@v3
+      - name: "Install Rust toolchain"
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2
+      - run: cargo install cargo-fuzz
```

Comment:
Does `cargo-bininstall` support `cargo-fuzz`? It could help to speed up the CI step

https://github.com/charliermarsh/ruff/blob/466719247bc68c0b608bc85be341c5aa5e2a5ec2/.github/workflows/benchmark.yaml#L81-L85

Reply:
I am unfamiliar with bininstall. I will try this :slightly_smiling_face:

Reply:
Hm, looking at the CI, it seems that none of the other installations in `ci.yaml` use binstall. Perhaps we should make this a separate PR instead.

Reply:
Good point. They probably should. But what I understand is that we probably want to remove the CI step for now anyway. Right?